content/chronograf/v1.8/tools/chronograf-cli/_index.md

```diff
@@ -102,15 +102,15 @@ chronograf [flags]
 | `--auth0-domain`        | Subdomain of your Auth0 client. Available on the configuration page for your Auth0 client.                                                                                                                                    | `$AUTH0_DOMAIN`        |
 | `--auth0-client-id`     | Auth0 client ID value for OAuth 2.0 support                                                                                                                                                                                   | `$AUTH0_CLIENT_ID`     |
 | `--auth0-client-secret` | Auth0 client secret value for OAuth 2.0 support                                                                                                                                                                               | `$AUTH0_CLIENT_SECRET` |
-| `--auth0-organizations` | Auth0 organization membership required to access Chronograf. Organizations are set using an organization key in the user’s `app_metadata`. Lists are comma-separated and are only available when using environment variables. | `$AUTH0_ORGS`          |
+| `--auth0-organizations` | Restricts authorization to  users from comma-separated list of specified Auth0 organizations. Optional. Organizations are set using an organization key in the user’s `app_metadata`.  | `$AUTH0_ORGS`          |
```

Comment:
I don't think this matches what the underlying issue outlines.  These are flags... and when using flags it implies having multiple flags.  

When using environment variables --> then the comma-separated list works.